crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -2035,3 +2047,45 @@ impl ExecutionComponents {
         self.metrics.clone()
     }
 }
+
+// Cache up to 20000 verified certs. We will need to tune this number in the future - a decent
```

Comment:
I think we can use a bigger number for now just to save time to tune this again, e.g. 100k?

Reply:
IMO we're good for now to test how it works and we can tune it in an upcoming PR